src/modules/accommodations/components/reviews/review-card.tsx

```diff
@@ -105,7 +105,7 @@ export function ReviewCard({ review }: ReviewCardProps) {
                 {userName}
               </Typography>
               <Typography variant="body2" color="textSecondary" mb={1}>
-                {userCountry} - from {selectedReview.user.createdAt}
+                {userCountry} - from {selectedReview.user.createdAt} Airbn user
```

Comment:
Why Airbnb? Our platform is StayHub. Also, the date is hard to read for a human. Please make it more readable. Something like "On StayHub since November 23" (if the year is the same as the current year) or "On StayHub since November 23, 2022" (if the year is not the same as current year).

Reply:
![image](https://github.com/user-attachments/assets/86025b09-3d9e-4dde-9c9b-7d250206f006)
Final version is like that.